### DISCUSSION TITLE
İş makinelerinin (vinç,kamyon vb.) "yardıma hazır" olanlarının haritalar yapılması

### DISCUSSION THREAD

**Discussion by sercangoger** [acikkaynak/deprem-yardim-projesi#39] [Ideas]
şuan iş makinesi olanların bilgileri mevcut. fakat konumları mevcut değil. bunların organize ve koordinasyonunu yönetmek adına haritalandırıp pinlemek doğru olacaktır. bu sayede nerede kim bekliyor gösterilecek AFAD VE STK için çok iyi olacaktır.

şuan makine kullanabilenlerin kaydı mevcutmuş oraya yeni bir özellik eklenebilir o yüzden,
bu fikri es geçelim. çünkü ben bunun için bir feature ekletebiliriz diye düşünüyroum. onun için issue olarak açıcam.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Ancak Legal ekibi ile değerlendirmemiz sonrasında "şartlı olarak onaylanabilir" kararına vardık.
Ek olarak Lojistik Projesinden yetkili kişilere fikir iletilmiştir.
"Legal olarak engel bulunmasa da lojistik kavramı altında değerlendirilebilecek bir konu. Rıza ve onay alınarak lojistik altında değerlendirilebilir."

**Reply by sercangoger** [reply]
teşekkürler konuyu takipteyim.

**Reply by sercangoger** [reply]
demin bir video izlerken bir enkaz zedenin söylediği şu cümle bile aaslında bu harita da iş makinelerinin gösterilmesi durumunu cok iyi anlatıyor.
"eğer bir kepçe olsaydı ben buradan 5-6 kişi çıkartırıdım" on kişiydik fakat kaldıramadık yapamadık diyor.
iki kişiyi çıkarabilmişler sadece. o yüzden
bunun önemini şimdi daha iyi anlıyorum. 
Harita iş makinelerinin gösterilmesi gerekli ve zorunlu

**Comment by agitrubard** [comment]
@sercangoger Selamlar,
Bu konuları da içeren bir proje için discussion oluşturmuştuk. İsterseniz oraya dahil edebiliriz, bu sayede fikirler birbirini ezmemiş olur ve fikir birliği ile birlikte ortaya çok iyi şeyler çıkabilir. 

[https://github.com/acikkaynak/afet-org/discussions/35](https://github.com/acikkaynak/afet-org/discussions/35)

Bu konu üzerine hem fikir olursak Discord üzerinden bana ulaşabilirsiniz;
`agitrubard#7048`
